db/BaseService/index.js

```diff
@@ -0,0 +1,57 @@
+const assert = require("assert");
+const { knex } = require("../helpers");
```

Comment:
This is bad for several reasons:

First, this is basically singleton pattern and should be pushed to the edges of the application where it is setup. This class should instead get the knex object from somewhere.
With the current design, knex (a long with its configuration) is loaded when the module is first required. This means that configuration of that module also has to happen at the same time or you would have to call into that same module later to change the configuration. This has to be done in tests also.

Second, it also means that all BaseServices have to use exactly the same knex configuration, which for a baseService might not that wise.